.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+-   id: cfn-module-generate
+    name: CloudFormation Module generate
```

Comment:
Can the name be changed into "AWS CloudFormation module generate command"?

Reply:
good call